File: Examples/DotNET/SDK/CSharp/Fields/GetMergeFieldNames.cs
using System;
using Aspose.Cloud;

namespace CSharp.Fields
{
    class GetMergeFieldNames
    {
        public static void Run()
        {            
            string inputfile = "doc-sample3.doc";

            // Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, string.Empty);

            // Get All Merge Field Names from a Word Document
            WordsFieldNamesResponse wordsFieldNamesResponse = Common.WordsService.WordsMailMerge.ReadDocumentFieldNames(inputfile, true, Common.FOLDER);

            Console.WriteLine("\n" + inputfile + " have " + wordsFieldNamesResponse.FieldNames.Names.Count + " Merge Field");
        }
    }
}

File: Examples/DotNET/SDK/VisualBasic/Fields/GetMergeFieldNames.vb
Imports Aspose.Cloud

Namespace Fields
    Class GetMergeFieldNames
        Public Shared Sub Run()
            Dim inputfile As String = "doc-sample3.doc"

            ' Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, String.Empty)

            ' Get All Merge Field Names from a Word Document
            Dim wordsFieldNamesResponse As WordsFieldNamesResponse = Common.WordsService.WordsMailMerge.ReadDocumentFieldNames(inputfile, True, Common.FOLDER)

            Console.WriteLine((Convert.ToString(vbLf) & inputfile) + " have " + wordsFieldNamesResponse.FieldNames.Names.Count + " Merge Field")
        End Sub
    End Class
End Namespace